### ISSUE TITLE
Focus should be on dialog header component -> p-confirmDialog for accessibility, Narrator directly jump to footer

### ISSUE THREAD

**Issue by prasadkhegade** [primefaces/primeng#9977]
If you have a PrimeNG PRO Support subscription please post your issue at;

https://pro.primefaces.org 

where our team will respond within 4 business hours.

If you do not have a PrimeNG PRO Support subscription, fill-in the report below. Please note that
your issue will be added to the waiting list of community issues and will be reviewed on a first-come first-serve basis, as a result, the support team is unable to guarantee a specific schedule on when it will be reviewed. Thank you for your understanding.

Current Queue Time for Review
Without PRO Support: ~8-12 weeks.
With PRO Support: 1 hour

**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
https://stackblitz.com/edit/primeng-confirmdialog-demo

**Current behavior**
In case of Narrator the focus directly goes to first button on footer.

**Expected behavior**
Focus should read in top down approach so that Narrator will start the reading from header -> content -> footer 

**Minimal reproduction of the problem with instructions**
Steps:

1. https://stackblitz.com/edit/primeng-confirmdialog-demo
2. Start Narrator in case of windows. Use appropriate accessibility tool in case of different os

**What is the motivation / use case for changing the behavior?**
This will bring better usability, visual representation in terms of accessibility for user

**Please tell us about your environment:**
Window, Linux

* **Angular version:** 5.X
9.1.11

* **PrimeNG version:** 5.X
9.1.0

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
Chrome Version 88.0.4324.104 (Official Build) (64-bit)
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
Typescript: 3.8.8

* **Node (for AoT issues):** `node --version` =   
12.18.2

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
